node/full_node_integration_test.go

```diff
@@ -169,25 +172,49 @@ func TestLazyAggregator(t *testing.T) {
 
 	require.NoError(err)
 
-	// delay to ensure it waits for block 1 to be built
-	time.Sleep(1 * time.Second)
+	// wait for 1st block to be built
+	require.NoError(testutils.Retry(300, 100*time.Millisecond, func() error {
+		fn, ok := node.(*FullNode)
+		assert.Equal(ok, true)
+		if fn.Store.Height() >= 1 {
+			return nil
+		} else {
+			fmt.Println("Retrying...")
+			return errors.New("awaiting first block...")
+		}
```

Comment:
nit: you don't need the else, you can keep this inline and I'd remove the println since it'll just spam the output. 

```suggestion
		if fn.Store.Height() >= 1 {
			return nil
		}
		return errors.New("Node is at blockheight 0")
```